Resources/ServerInfo/Guidebook/NewPlayer/Controls/Radio.xml

```diff
@@ -0,0 +1,65 @@
+<Document>
+# Speech and Text Channels
+Talking is a key part of this game, both literally and fuctionally. You can press "T" to jump to the text box.
```

Comment:
```suggestion
Talking is a key part of the Space Station 14, both literally and functionally. You can press "T" to jump to the text box.
```

Can you elaborate on the "literally and functionally" part?

Reply:
the idea was literally - you talk to play the game - and functionally - talking is a big part code wise - but i see how it's ambiguous

Reply:
i made a commit to fix this, i'm gonna rip the bandaid off instead